sql/table.h

```diff
@@ -731,26 +733,30 @@ struct TABLE_SHARE
   uint16 row_start_field;
   uint16 row_end_field;
   uint32 hist_part_id;
-  List<void> free_parts;
-  bool free_parts_init;
+  Vers_field_stats** stat_trx;
+  ulonglong stat_serial;
```

Comment:
In that case it would be great to have a comment with a list of variables this rwlock guards.